pom.xml

```diff
@@ -395,6 +395,7 @@
         <carbon.identity.package.import.version.range>[5.0.0, 6.0.0)</carbon.identity.package.import.version.range>
         <javax.servlet-api.version>3.0-alpha-1</javax.servlet-api.version>
         <javax.servelet.jstl.version>1.2</javax.servelet.jstl.version>
```

Comment:
Can we remove this property if it is no longer needed?

Reply:
Yes, it is safe to remove.